.circleci/config.yml

```diff
@@ -536,6 +542,7 @@ jobs:
       # to the sources due to path mapping.
       - run: yarn ngcc --error-on-failed-entry-point --no-tsconfig
       - *slack_notify_on_failure
+      - *only_main_branch_filter
```

Comment:
Disclaimer: I don't actually know if this will work as expected, I'm basing it on how we've dealt with similar situations in other places. As far as I can tell, there's no good way of testing it without merging in the changes.